src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -81,9 +126,85 @@
  *   &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_ELSE, METHOD_DEF&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ *public class Test {
  *
- * @since 3.0
+ *  public void test() {
+ *
+ *    if (foo) {
+ *      bar();
+ *    } else { bar(); }   // violation, right curly must be alone
```

Comment:
`// violation, right curly must be alone on line`

Reply:
done